### ISSUE TITLE
[python] async is a keyword in python 3.7

### ISSUE THREAD

**Issue by kevinresol** [HaxeFoundation/haxe#7445]
```
> python3 bin/python/tests.py
  File "bin/python/tests.py", line 1733
    ret = tink_core__Future_Future_Impl_.async(_hx_local_1,lazy).map(tink_core_Outcome.Success)
                                             ^
SyntaxError: invalid syntax
```

**Comment by kevinresol** [comment]
Sorry I just realize this is already fixed in haxe dev

https://github.com/HaxeFoundation/haxe/commit/1c0a4c3e833ba151cb2d27eae28988c609212d7c

**Comment by cancerberoSgx** [comment]
This issue happens for me in 3.4.7 - Just want to know if it was injected again or it will be only fixed in haxe 4 ? Thanks

**Comment by Gama11** [comment]
The fix happened after the release of 3.4.7, and I don't think any more 3.4 patch releases are planned.
